EIPS/eip-3220.md

```diff
@@ -0,0 +1,121 @@
+
+## Motivation
+
+EIP-155 was proposed by Vitalik to provide a unique identifier to a blockchain to provide simple relay attack protection.  This specification defines an integer for Chainid for a blockchain and sign the chainid into a transaction data and hence present attackers to send same transaction to different blockchains. This specification will require blockchains to define a chainid and register the chainid in a public repository.
+
+The challenge of using an integer for chainid is that it is not broad enough to cover all blockchains and it does not prevent different blockchains using the same chainid.  Also, it does not address the issue for two forked blockchains having the same chainid.
+
+Hence there is need for a more robust blockchain identifier that will overcome these drawbacks, especially for crosschain operations where multiple chains are involved. A blockchain identifier (crosschain id) should be unique and satisfy the following requirements:
+
+should provide identification, description, and discovery of blockchains.
+should provide unique identification of each blockchain in the crosschain service ecosystem.
+should provide descriptions for a blockchain identities such as chainId, name, type, consensus scheme etc.
+should provide discovery mechanism for supported blockchains and also for new blockchains in the ecosystem.
+should provide a mechanism for a joining blockchain to register to the ecosystem.
+should provide a mechanism for a blockchain to edit properties or unregister from the crosschain ecosystem.
+should provide a mechanism to get some critical information of the blockchain
+should provide a mechanism to differentiate an original blockchain and a forked blockchain
+should provide a mechanism to verify a chainid without external registration service
+
+## Specification
+
+The section specifies a) The crosschain id definition b) the registration smart contract the defines metadata to be associated with a crosschain id; and c) the basic functions to parse and process a crosschain id.
+
```

Comment:
```suggestion
```
I think we can remove this as it doesn't really add much value.  The style of an EIP specification should not be like an essay where you have an introduction, body, conclusion.  Just focus on the technical specification only.